scripts/parser/analyze.js

```diff
@@ -45,7 +55,7 @@ const lspPluginConfigOptions = tsConfig?.config?.compilerOptions?.plugins?.find(
 
 if (!lspPluginConfigOptions?.parser) {
   console.error(`Cannot get parser config from tsconfig found at: "${tsconfigPath}"`);
-  process.exit(2);
+  process.exit(3);
```

Comment:
GitHub eslint action:

No magic number: 3

Reply:
https://github.com/genesiscommunitysuccess/custom-elements-lsp/pull/52/commits/5c15bdc279062315aa658570464f704f1f2bb78c
Odd that it was happy with magic numbers for 1 and 2 exit codes? 😕